### PR TITLE
Update UG to reflect bug fix in find

### PR DESCRIPTION
**Pull request by gnahzeus**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -383,7 +383,7 @@ Fields supported in `find`:
 - Class Date `dt/`
 - Tag `t/`
 
-<div markdown="span" class="alert alert-danger">❗ **Caution:** Only one field can be searched at once.
+<div markdown="span" class="alert alert-danger">❗ **Caution:** Only one prefix should be used in a find command at a time.
```
Comment (Echomo-Xinyu) [inline docs/UserGuide.md:386]: As I raised in the issue #307 , I think we need to further clarify about the sentence here.

Comment (Echomo-Xinyu) [reply docs/UserGuide.md:386]: The issue #307 is not planned to be implemented.

**Review by Echomo-Xinyu — APPROVED** [review]
LGTM! It would be great if you can fix the typo in line 505 `dt/javascript` to be `t/javascript` in this PR as well.

**Comment by seanflyyy** [comment]
close #273

**Pull request merged**